scripts/download.js

```diff
@@ -12,7 +12,7 @@ const timeout = 120000
 
 export default async function downloadNetfile({ agencyId, year }) {
     const downloadPageUrl = `https://public.netfile.com/pub2/Default.aspx?aid=${agencyId}`
-    const browser = await puppeteer.launch({ timeout })
+    const browser = await puppeteer.launch({ headless: "new", timeout })
```

Comment:
For whatever reason, Github actions doesn't like running puppeteer with `headless: "new"` - not totally sure what the issue was but it went away when I removed this. Maybe we should just deal with the deprecation notice for now and see if stability improves?

Reply:
I will revert this change.

The warning says `In the near feature headless: true will default to the new Headless mode [...] Consider opting in early by passing headless: "new"` . It should not give us a warning once it updates in the future to a newer version.